problems/water/water.adoc

```diff
@@ -52,18 +52,23 @@ check50 cs50/2017/x/water
 ----
 
 == Staff Solution
-
+Not sure how your solution should behave?
+Run our example by calling '~cs50/pset1/water'. Assumed that the underlined text is what some user has typed.
 [source]
 ----
-~cs50/pset1/water
+$ [underline]#~cs50/pset1/water#
+Minutes: [underline]#10#
+Bottles: 120
 ----
 
 == Hints
 
-* You can use the function `get_int` to prompt the user to input an integer value, as per the below:
+* 'cs50.h' has a helper function `get_int()` that prompts the user to input an integer value:
```

Comment:
What is the need for this change in wording? It seems like adding the `#include <cs50.h>` line below should be sufficient to indicate that the header file must be included.

Reply:
It is easy to overlook where `get_int()` is from and why we need to include the header file. The extra wording is descriptive and instructive, giving a better context in why we need to `#include <cs50.h>`